sklearn/tests/test_dummy.py

```diff
@@ -8,7 +8,6 @@
 from sklearn.utils._testing import assert_array_equal
 from sklearn.utils._testing import assert_array_almost_equal
 from sklearn.utils._testing import assert_almost_equal
-from sklearn.utils._testing import assert_raises
```

Comment:
Please revert this change because it is unrelated with the the issue that you want to do here.
It is true for the entire file.